docs/reference/containerized-execution.mdx

```diff
@@ -104,3 +95,30 @@ The federates `foo` and `bar`, the images of which have already been built, can
 docker run -t --rm --network lf foo
 docker run -t --rm --network lf bar
 ```
+
+## Configuration Options
+You can further customize the generated Docker file through the `docker` target property. Instead of just enabling Docker support using `true`, specify configuration options in a dictionary.
+
+### Select a Base Image
+You can specify a base image (`FROM`) as follows:
+
+```lf-c
+target C {
+    docker: {FROM: "alpine:latest"}
+}
+```
+
+This specifies that the base image is the latest version of `alpine`, a very small Linux. In fact, `alpine:latest` is the default value for this option, so you only need to specify this option if you need something other than `alpine:latest`.
+
+### Select an RTI
+By default, an image is pulled from DockerHub that contains the RTI. An alternative image can be specified using the `rti-image` entry as follows:
+
+```lf-c
+target C {
+    docker: {rti-image: "lflang/rti:latest"}
+}
+```
+
+:::note
+To use a local image that is based on source of the RTI provided by `lfc`, use `rti-image: "lflang/rti:rti"`.
+:::
```

Comment:
This does not explain what `rti-image: local` means.  Also, what is meant by "source of the RTI provided by `lfc`"?  Does `lfc' provide source?  Also, the syntax `lflang/rti:rti` is pretty arcane.  What does this mean?